.github/workflows/import-analysis.yml

```diff
@@ -0,0 +1,63 @@
+name: Import analysis
+
+on:
+  pull_request:
+
+jobs:
+  import_analysis:
+    name: "Bootstrap import analysis"
+    runs-on: ubuntu-latest
+    permissions:
+       pull-requests: write
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+          path: dd-trace-py
+
+      - name: Install Python 3.12
+        uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
+        with:
+          python-version: '3.12'
+
+      - name: Install rust toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@9399c7bb15d4c7d47b27263d024f0a4978346ba4 # v1.11.0
+        
+      - name: Install apt dependencies
+        run: sudo apt-get update && sudo apt-get install -y --no-install-recommends build-essential
+
+      - name: Collect import data from PR
+        run: |
+          python -m venv .venv-pr
+          source .venv-pr/bin/activate
+          .venv-pr/bin/python -m pip install ./dd-trace-py
+          .venv-pr/bin/python -c "import ddtrace.auto"
+          for n in {0..500}; do
```

Comment:
is `500` necessary? I see the job takes around 16 minutes to run, is it stable enough with 100 or 200 runs as well?

Reply:
oh, I suppose building time is probably a good chunk of it....

Reply:
Yeah build time it what kills this. And yes, we need at least 400ish samples because that about the number of independent import stacks we currently observe. In fact I'm thinking of bumping this up to 1000